blog-posts/return-to-office-interview.md

```diff
@@ -0,0 +1,142 @@
+
+**Eric:** Ruhe. Zu Hause fällt es vielen einfacher, in den Tunnel zu kommen, den es braucht um konzentriert an komplexen Themen arbeiten zu können.
+Ich glaube aber auch, dass es einfach Bequemlichkeit ist bzw. “Convenience”, hört sich schöner an (lacht). Ich muss mich nicht erst morgens duschen, waschen, anziehen, fertig machen und in die Arbeit commuten.
+
+**Gholam:** (lacht) Jetzt überträgst du von dir auf andere!
+
+<br />
+
+![Felix im virtuellen Office](images/return-to-office-interview-felix.png)
+<small>Auch virtuell wurde bei Satellytes stets auf ein professionelles Erscheiungsbild geachtet!</small>
+
+<br />
+
+**Eric:** Nein, aber mal ehrlich ... Ich kann morgens direkt den Rechner aufklappen und bin voll im Fokus und kann dann Pause und meine Morgenroutine machen, wenn die Konzentration das erste Mal nachlässt. Wenn ich vom Büro aus arbeite, ist mein Morgen weniger flexibel planbar.
+Allgemeiner gefasst, kann man sagen: Die Remote-Arbeit erlaubt jedem, die eigene Routine zu finden, die am besten geeignet ist um gut und effizient arbeiten zu können.
+
+**Gholam:** Darüber hinaus glaube ich, dass vor allem die freie Ortswahl ein Grund ist, dass die Leute Gefallen an der Remote-Arbeit gefunden haben. Arthur z.B. hatte vor der Pandemie jeden Tag 70 Kilometer zur Arbeit gehabt. Das sind bis zu zwei Stunden, die täglich verloren gehen. Nach zwei Jahren funktionierendem Home-Office, stellt sich dann natürlich die Frage: Ist das wirklich jeden Tag nötig?
+Damit einher geht die bessere Vereinbarkeit von Familie und Beruf. Ich kann mich noch an Projekte erinnern, wo es Pflicht war spätestens um 9:00 Uhr vor Ort zu sein und in die Tasten zu hauen.
+In der Zeit, musste ich davor aber auch jeden Morgen meinen Sohn zur Kita bringen. Das hieß dann morgens mit S-Bahn, Bus und Fussweg von Milbertshofen im Norden nach Untermenzingen in die Kita. Danach dann mit Bus und S-Bahn zur Arbeit. Das war schon ein brutaler Stress, den ich jeden Morgen hatte. Praktisch hieß das, dass ich spätestens um 7:00 Uhr das Haus verlassen musste, damit ich auch garantiert um 9:00 Uhr pünktlich im Büro sein konnte.
+Wenn so ein Zwang wegfällt, ist man ganz automatisch auch entspannter und kann mehr Energie in die Arbeit investieren.
+
+**Was plant ihr für die “Rückkehr zum Arbeitsplatz”? Wird es Vorgaben geben, was den Arbeitsort angeht?**
+
+**Gholam:** Das werden wir unseren Mitarbeitern komplett frei stellen. Wir wollen da keine festen Vorgaben machen. Von Vollzeit Remote, Vollzeit Büro und allem was dazwischen denkbar ist, werden wir den Mitarbeitern die Freiheit geben, ihren Arbeitsrhythmus so zu planen, dass sie gut arbeiten können.
+
+**Eric:** Als einzige Ausnahme sehen wir hier Senior- bzw Manager-Aufgaben. Ein Beispiel: Zum Mentoring gehört ja nicht nur das Weitergeben von Hard Skills, sondern auch von Soft Skills und das funktioniert in persona einfach erfahrungsgemäß besser als remote.
```

Comment:
🔴
I would disagree here until clarified and I think it's difficult to speak about a wish/exception here in the c-role as Eric. That's why I would appreciate to remove this part or shape it differently and more clearly as an opionion

Reply:
I've updated the paragraph to it's previous state. Please check in again to see if you agree with the sentiment reflected there. 
I personally think that the sentiment given by Eric is worth publishing, since it reflects his beliefs. 
But if this still doesn't sound right to you, we can remove it.

Reply:
I checked the new paragraph. Thanks sounds better like that.